libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -955,6 +955,70 @@ def SetTags(self, new_tags: List[str]) -> None:
     response = request.execute()
     self.BlockOperation(response, zone=self.zone)
 
+  def GetPowerState(self) -> str:
+    """
+    Gets the current power state of the instance.
+
+    As per https://cloud.google.com/compute/docs/reference/rest/v1/instances/get
+    this can return one of the following possible values: PROVISIONING, STAGING,
+    RUNNING, STOPPING, SUSPENDING, SUSPENDED, REPAIRING, and TERMINATED
+    """
+    return str(self.GetOperation()['status'])
+
+  def Stop(self) -> None:
+    """
+    Stop the instance.
+    """
+
+    try:
+      logger.info('Stopping instance "{0:s}"'.format(self.name))
```

Comment:
the `logger.info` statements can go outside the `try/except` block, as there's no chance they'd raise a `HttpError`